plutus-ledger-api/src/PlutusLedgerApi/V1/Data/Value.hs

```diff
@@ -232,18 +232,18 @@ newtype AssetClass = AssetClass {unAssetClass :: (CurrencySymbol, TokenName)}
   deriving (Pretty) via (PrettyShow (CurrencySymbol, TokenName))
 
 instance HasBlueprintSchema AssetClass referencedTypes where
-  {-# INLINEABLE schema #-}
+  {-# INLINABLE schema #-}
```

Comment:
FYI: `fourmolu` also does this kind of a normalisation.

Reply:
> FYI: fourmolu also does this kind of a normalisation.

Good.  I don't have strong opinions about how it should be spelled, but I noticed that in this file both forms were used, and I felt compelled to make them all the same.